pymilvus/milvus_client/index.py

```diff
@@ -37,8 +40,10 @@ def __eq__(self, other: None):
 
 
 class IndexParams:
-    def __init__(self):
+    def __init__(self, field_name: Optional[str] = "", **kwargs):
```

Comment:
Optional mean None or str, if it has a default value "", just use `filed_name: str = ""`

Reply:
Or `field_name: Optional[str] = None`